3Dmol/3dmol.js

```diff
@@ -157,6 +157,7 @@ $3Dmol.getbin = function(uri, callback) {
     $.ajax({url:uri, 
         type: "POST",
         dataType: "binary",
+        async: "false",
```

Comment:
This can't be left in.  Add an optional argument (default to async) to enable synchronous communication.